python_utils/rec.py

```diff
@@ -0,0 +1,88 @@
+import sys
+import matplotlib.pyplot as plt
+import math
+import numpy as np
+import collections
+
+#Que definition
+perm_amp = collections.deque(maxlen=100)
+perm_phase = collections.deque(maxlen=100)
+#Set subcarrier to plot
+Subcarrier=44

```

Comment:
Thanks for extracting this out into a variable. Can you make it lowercase to keep with the common python variable-styling.